core/base/src/TROOT.cxx

```diff
@@ -930,7 +930,7 @@ TROOT::~TROOT()
       SafeDelete(fGlobals);
       if (fGlobalFunctions) fGlobalFunctions->Delete();
       SafeDelete(fGlobalFunctions);
-      fEnums->Delete();
+      fEnums.load()->Delete();
```

Comment:
My bad, I was off by one level of indirection :)
And indeed, elsewhere in the code I have used
```
(*fEnums).Delete();
```
thanks.